### ISSUE TITLE
Create BaseCommand class

### ISSUE THREAD

**Issue by tobiasraabe** [tobiasraabe/otree_virtual_machine_manager#4]
* oTree Ubuntu Server Manager version: 0.1.0
* Python version: 3.6
* Operating System: Ubuntu

### Description

A much more sophisticate approach regarding commands would be to build ``class BaseCommand`` which could look like this:

````
# commands/base.py
"""The base command"""

class BaseCommand(object):
    """A base command."""
 
    def __init__(self, options, *args, **kwargs):
        self.options = options
        self.args = args
        self.kwargs = kwargs

   def requirements_check():
        pass
 
    def run(self):
        raise NotImplementedError('You must implement the run() method yourself!')
````

Additional ressources:
- https://stormpath.com/blog/building-simple-cli-interfaces-in-python
- https://github.com/django/django/tree/master/django/core/management
- https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/

**Comment by cataclysmic** [comment]
Do we still need this?

**Comment by tobiasraabe** [comment]
No, this will be closed with the commit message when I merge #102 .
